### PR TITLE
build(deps): upgrade ovh-ui-angular to v3.2.2

### PR DESCRIPTION
**Pull request by frenautvh**
See : MANAGER-2687

**issue** : errors in js console due to oui-field component

**fix** : fix oui-field component, bump ovh-ui-angular to version 3.2.2

### CODE REVIEW

File: packages/manager/apps/freefax/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "oclazyload": "^1.1.0",
     "ovh-angular-ui-confirm-modal": "^1.0.2",
     "ovh-api-services": "^6.23.0",
-    "ovh-ui-angular": "^3.2.1",
+    "ovh-ui-angular": "^3.2.0",
```
Comment (marie-j) [inline packages/manager/apps/freefax/package.json]: Should be `3.2.2` right ?

**Comment by antleblanc** [comment]
@frenautvh Could you please resolve existing conflicts?

Thanks!

File: packages/manager/apps/freefax/package.json (hunk 1 above)
Comment (frenautvh) [reply packages/manager/apps/freefax/package.json]: i prefer ^3.2.0 since it includes 3.2.2 and further future patches 3.2.x, what do you think?

Comment (jleveugle) [reply packages/manager/apps/freefax/package.json]: Depends on your needs!

Maybe you need a fix in ovh-ui-angular which means that you need at least the version 3.2.2. `^3.2.2` is a good compromise ;D

**Comment by frenautvh** [comment]
@antleblanc done!

**Pull request merged**